pkg/csi_driver/multishare_controller.go

```diff
@@ -590,6 +625,9 @@ func (m *MultishareController) generateCSICreateVolumeResponse(instancePrefix st
 	if m.driver.config.FeatureOptions.FeatureLockRelease.Enabled {
 		resp.Volume.VolumeContext[attrSupportLockRelease] = "true"
 	}
+	if m.featureMaxSharePerInstance {
+		resp.Volume.VolumeContext[attrMaxShareSize] = strconv.Itoa(int(maxShareSizeSizeBytes))
```

Comment:
what is this used for? i assume in the future when `ControllerExpandVolume` is called this context will be used to determine if it's pass limit?

Reply:
Correct. ExpandVolume does not have the volume context, so we will infer it from the PV spec.csi.volumecontext